dan_layer/consensus/src/hotstuff/on_propose.rs

```diff
@@ -694,7 +695,7 @@ where TConsensusSpec: ConsensusSpec
                     "⚠️ Transaction is LOCAL-ONLY EARLY ABORT, proposing LocalOnly({}, ABORT)",
                     tx_rec.transaction_id(),
                 );
-                tx_rec.set_local_decision(Decision::Abort);
+                tx_rec.set_local_decision(Decision::Abort(AbortReason::EarlyAbort));
```

Comment:
This is due to local inputs not existing or being down
```suggestion
                tx_rec.set_local_decision(Decision::Abort(AbortReason::LockInputsFailed));
```